cmd/check.go

```diff
@@ -275,7 +275,7 @@ func (r recocheckDep) makeVirtualGoPath() error {
 			switch pErr.Err.Error() {
 			case os.ErrNotExist.Error():
 				return nil
-			case "no such file or directory":
+			case "No such file or directory.":
```

Comment:
This should be kept as it was, it's matching a string output by another function. The other function is part of the os library so cannot be changed easily.